Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -481,6 +481,10 @@ def audit_options
 
       next unless @strict
 
+      if o.name == "universal"
+        problem "macOS has been 64-bit only since 10.6 so universal options are deprecated."
```

Comment:
You could do '64-bit–only' (first a normal hyphen/hyphen-minus, then an en dash,) per [section 1.2.1.3 of Wikipedia's article on dashes](https://en.wikipedia.org/wiki/Dash#Attributive_compounds), but, as noted there near the bottom, that can be somewhat controversial and should only be used to disambiguate when necessary.  My gut feeling says that said disambiguation would increase readability, even if only slightly, here, though maybe I'm just hungry…; in any case, it's your guys's call to make, and probably in a future PR, as well, as you closed this one.  Feel free to ignore my nit-picking if you wish.